src/python/pants/reporting/json_reporter.py

```diff
@@ -61,7 +61,7 @@ def close(self):
     safe_file_dump(
```

Comment:
Can you just delete your override of `close` now? The base class noops, and that would be just fine. You could drop `self._report_path` to and the setup code that entails.